recipes/adacpp/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = 'ada-cpp' %}
+{% set version = '0.1.0' %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  - url: https://github.com/Krande/adacpp/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 499fb878bb3c1d1bd2bd219f747f6e1f41883c6f63e81e192af204664e8908f2
+    patches:
+      - patches/remove_hardcoded_osx_sysroot_and_deployment_target.patch
+
+build:
+  skip: True  # [py<310]
+  number: 0
+  string: py{{ CONDA_PY }}_{{ variant }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+  run_exports:
+    - {{ pin_subpackage(name, exact=True) }} *{{ variant }}*
+
+requirements:
+  build:
+    - {{ stdlib("c") }}
+    - "{{ compiler('c') }}"
+    - "{{ compiler('cxx') }}"
+    - "{{ cdt('mesa-libGL-devel') }}"  # [unix] provides OpenGL development libraries
+    - cmake
+    - ninja  # [win]
+    - make  # [unix]
+
+  host:
+    - python
+    - occt * *{{ variant }}*
+    - ifcopenshell
+    - cgal-cpp
+    - scikit-build-core
+    - nanobind
+    - cli11
+    - tinygltf
+    - gmsh
+    - pip
+
+  run:
+    - python
+    - "{{ pin_compatible('gmsh', exact=True) }}"
```

Comment:
Do you really need the exact same build of gmsh? Normally this is pinned to some patch/minor level.

Reply:
Yea, I think you're right. I'll pin this to the minor level